pkg/utils/metadata.go

```diff
@@ -14,18 +14,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
LOL github thinks this file is a rename but it really isn't. Git tracks renames by comparing how similar files are, and these were just similar enough it seems.

Reply:
Oops!